usr/src/uts/armv8/os/fakebop.c

```diff
@@ -1015,6 +1059,22 @@ build_firmware_properties(struct xboot_info *xbp)
 	}
 }
 
+/*
+ * fake up a boot property for deferred early console output
+ * this is used by both graphical boot and the (developer only)
```

Comment:
This seems like it's saying something hardware-specific, and I'm not sure what it is.

We definitely want serial consoles as an option long-term

Reply:
This simply allocates the early buffer for console types that only attach later (USB-Serial and graphical). I've looked around, and I can't find any trace of how those things are supposed to work, but the changes to support them at this level seemed pretty straightforward.

Reply:
Seems a reasonable thing to go with for now